ambry-quota/src/main/java/com/github/ambry/quota/AmbryQuotaManager.java

```diff
@@ -147,6 +153,19 @@ public QuotaConfig getQuotaConfig() {
     return quotaConfig;
   }
 
+  /**
+   * Notify {@link QuotaSource}s about creation of new Ambry {@link Account} or {@link com.github.ambry.account.Container}.
+   * @param updatedAccounts {@link Collection} of {@link Account}s updated.
+   */
+  protected void onAccountUpdateNotification(Collection<Account> updatedAccounts) {
+    requestQuotaEnforcers.stream().map(QuotaEnforcer::getQuotaSource).forEach(quotaSource -> {
```

Comment:
there are a couple things that can be done to improve this stream operation.
1. move `updatedAccounts.stream()` out so we only have to do it once.
2. filter out the null quota source since there is no guarantee `QuotaEnforcer` will return a non-null `QuotaSource`.